nx/lib/nx.ex

```diff
@@ -5245,6 +5245,38 @@ defmodule Nx do
     impl!(tensor).conjugate(%{tensor | type: Nx.Type.to_complex(tensor.type)}, tensor)
   end
 
+  @doc """
+  Calculates the complex phase angle of each element in the tensor.
+  If $z = a + bi = r e^\\theta$, $phase(z) = atan2(b, a)$
+
+  ## Examples
+       iex> Nx.phase(Complex.new(1, 2))
+       #Nx.Tensor<
+         f32
+         1.1071487665176392
+       >
+       iex> Nx.phase(1)
+       #Nx.Tensor<
+         f32
+         0.0
+       >
+       iex> import Nx, only: [sigil_V: 2]
+       iex> Nx.phase(~V[1+2i -2+1i])
+       #Nx.Tensor<
+         f32[2]
+         [1.1071487665176392, 2.677945137023926]
+       >
+  """
+  def phase(tensor) do
```

Comment:
```suggestion
  defn phase(tensor) do
```

You'll need to import this

Reply:
I don't understand why we are making an exception for `phase` and using defn?